docs/audits/deprecated-api-usage.md

```diff
@@ -0,0 +1,21 @@
+# Avoid deprecated GPT APIs
+
+## Overview
+
+This audit ensures that deprecated Google Publisher Tag (GPT) APIs are not used.
```

Comment:
Nit: maybe avoid the passive voice?

How about: "This audit ensures that the page does not use any deprecated Google Publisher Tag (GPT) APIs."

Reply:
Done.